### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by qianz2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +26,7 @@ If you are looking for a GitHub Action to deploy your Azure Web App, consider us
 
 The definition of this GitHub Action is in [`action.yml`](./action.yml).
 
-Now we have a beta version [`v2-beta`](https://github.com/Azure/appservice-build/tree/v2-beta) released. It's using a [`new image specifically for GitHub Actions`](https://github.com/microsoft/Oryx/blob/master/images/build/GitHubActions.Dockerfile
+Now we have a beta version [`v2-beta`](https://github.com/Azure/appservice-build/tree/v2-beta) released. It's using a [`new dockerfile for GitHub Actions`](https://github.com/microsoft/Oryx/blob/master/images/build/GitHubActions.Dockerfile
```
Comment (kichalla) [inline README.md]: Should we remove references to `beta`?

File: README.md [hunk 2]
```diff
@@ -54,6 +57,11 @@ jobs:
 
       - name: Building web app
         uses: azure/appservice-build@v1
```
Comment (kichalla) [inline README.md]: I would suggest to change all references from `v1`  to `v2` since we want to promote `v2` going forward

File: README.md [hunk 3]
```diff
@@ -82,6 +95,23 @@ jobs:
 
 The following variable should be replaced in your workflow:
 
+- `<PLATFORM_NAME>`
+    - Programming platform used by the web app that's being deployed, **optional**.
+
+- `<PLATFORM_VERSION>`
+    - Version of programming platform used by the web app, **optional**.
+    - Default version of each platform if not set:
```
Comment (kichalla) [inline README.md]: This is not entirely correct. If a user does not set a version explicitly, we detect the version from the source directory. If a version was not determined from the source files, then we fall back to default version.

**Review by kichalla — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (qianz2) [reply README.md]: Good point, thanks. Will specify this.

**Pull request merged**